app/src/main/java/com/example/playlistmaker/player/ui/view_model/FragmentPlayerViewModel.kt

```diff
@@ -86,6 +92,34 @@ class PlayerViewModel(
         return likeIndicator
     }
 
+    fun loadPlaylists() {
+        viewModelScope.launch {
+            playlistsInteractor
+                .playlistGet()
+                .collect { playlists ->
+                    processResult(playlists)
+                }
+        }
+    }
+
+    private val stateLiveData = MutableLiveData<PlaylistsState>()
+    fun observeState(): LiveData<PlaylistsState> = stateLiveData
+    private fun processResult(playlists: List<Playlist>) {
+        if (playlists.isEmpty()) {
+            stateLiveData.postValue(PlaylistsState.Empty(R.string.nothing_in_favourite))
```

Comment:
🍏 Рекомендую переименовать строку, так как она уже используется не только с избранным
```suggestion
            stateLiveData.postValue(PlaylistsState.Empty(R.string.nothing_in_mediateka))
```